src/components/explore-button.tsx

```diff
@@ -0,0 +1,50 @@
+import { inject, observer } from "mobx-react";
+import * as React from "react";
+import { BaseComponent, IBaseProps } from "./base";
+
+import "./explore-button.sass";
+import { Spaces } from "../models/ui";
+
+interface IProps extends IBaseProps {
+  space: Spaces;
+  title: string;
+}
+interface IState {}
+
+@inject("stores")
+@observer
+export class ExploreButtonComponent extends BaseComponent<IProps, IState> {
+
+  public render() {
+    const {ui} = this.stores;
+    let classSuffix: string = ((ui.investigationPanelSpace === this.props.space) ? "active" : "clickable");
+    switch (this.props.space) {
+      case "populations":
```

Comment:
I might prefer a map or just using `space + suffix` here. At the very least, I'd include a `default` case if you go for a switch statement

Reply:
Unifying the CSS and space names is probably the easiest (as long as we're om with the assumption that the CSS class will match the space name.  I am.  Will change.

Reply:
This is fixed.